Expensify/App
# Consolidate NVP requests into single request

cc @stitesExpensify 

### Details


### Fixed Issues
$ https://github.com/Expensify/App/issues/4883

### Tests

**Note:** Going to keep the testing minimal here as I'm assuming if a few of these NVPs work then the rest of them should work as well. I didn't bother to set the private NVPs in this case, but I think there should be no issue with them at all since they will be handled much like the non private NVPs.

1. Added the following public NVPs via `expensify.com.dev` JS console so that they would appear when fetched
```js
NVP.set('expensify_migration_2020_04_28_RunKycVerifications', '2020-01-13')
NVP.set('expensify_ACHData_throttled', '2021-09-09 18:17:58');
```
2. Navigated to `https://localhost:8080/bank-account` with the network tab open to capture http response data
3. Located the `Get` command
4. Verified the NVPs appear
5. Verified the logic in the API handler works since we are handling the error about throttled account
![2021-09-09_12-19-51](https://user-images.githubusercontent.com/32969087/132770531-3a9b4c50-f958-46d7-927d-d8c11fd3974a.png)

### QA Steps

1. Navigate to `/bank-account` and verify the initial screen loads without any issue

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
